CIP-XXXX/README.md

```diff
@@ -0,0 +1,46 @@
+---
+CIP: ?
+Title: CIP-30 ext: Account public key
+Category: Wallets
+Status: Proposed
+Authors:
+    - Ola Ahlman <ola.ahlman@tastenkunst.io>
+    - Andrew Westberg <andrewwestberg@gmail.com>
+Implementors: [Eternl, newm-chain]
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/?
+Created: 2023-09-03
+License: CC-BY-4.0
+---
+
+## Abstract
+This CIP extends [CIP-30 (Cardano dApp-Wallet Web Bridge)](https://cips.cardano.org/cips/cip30/) to provide an additional endpoint for dApp to get the extended account public key from a connected wallet.
+
+## Motivation: why is this CIP necessary?
+Normally it's up to the wallet to handle the logic for utxo selection, derived addresses etc through the established CIP-30 api. Sometimes however, dApp needs greater control due to subpar utxo selection or other specific needs that can only be handled by chain lookup from derived address(es). This moves the control and complexity from wallet to dApp for those dApps that prefer this setup. A dApp has better control and can make a more uniform user experience. By exporting only the account public key, this gives read-only access to the dApp.
+
+## Specification
+A new endpoint is added namespaced according to this cip extension number that returns the connected account extended public key as [cbor\<T>](https://cips.cardano.org/cips/cip30/#cbort) defined in CIP30.
+
+### 1. api.cip????.getAccountPub(): Promise\<cbor\<Bip32PublicKey>>
+
+Errors: APIError
+
+Returns hex-encoded string representing cbor of extended account public key. Throws APIError if needed as defined by CIP30.
+
```

Comment:
How should this proposal be versioned? see [specification from CIP-0001](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0001/README.md#structure).

Reply:
🤔 ... any recommendation for wording on this? 
I'm open to any suggestion really.

Reply:
something like

"This CIP is not to be versioned using a traditional scheme, rather if any large technical changes are required then a new proposal must replace this one. Small changes can be made if they are completely backwards compatible with implementations, but this should be avoided."